core/src/main/java/binnie/core/BinnieCore.java

```diff
@@ -189,11 +188,6 @@ public void init(final FMLInitializationEvent evt) {
 		GameRegistry.registerTileEntity(TileEntityMetadata.class, "binnie.tile.metadata");
 	}
 
-	@Mod.EventHandler
-	public void postInit(final FMLPostInitializationEvent evt) {
-		super.postInit(evt);
```

Comment:
This method is actually needed. It is a very bad design, but this one is receiving the event because it is marked as `@Mod.EventHandler`, and it passes it up to `super` which is not marked as an event handler.